packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -142,10 +152,11 @@ export const ValuedReferenceAutocomplete = ({
           helperText={widget.diagnostics[0]?.message}
           InputProps={{
             ...params.InputProps,
+            style: { paddingRight: theme.spacing(10) },
```

Comment:
And why 10?

Reply:
10 corresponds to the offset required to prevent the values from being displayed below the buttons.
I will add a comment